modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/DefinitionOfTerms.tsx

```diff
@@ -190,7 +174,9 @@ export function DefinitionOfTerms({baseResourceURL}: IProps) {
 					)}
 				</AutoComplete>
 
-				<div id="lfr-notification-web__definition-of-terms-render-fds" />
+				<div id="lfr-notification-web__definition-of-terms-render-fds">
```

Comment:
Hey @rebsilva, since we no longer need the `render` function you removed in lines `137-147`, do we really need this `div`?

Reply:
Thanks for the explanation @rebsilva, I think this name is good for the `id`